sfepy/examples/diffusion/Non_linear_general_poisson_equation.py

```diff
@@ -0,0 +1,104 @@
+r"""
+Example usage of the non linear diffusion and non linear volume terms.
```

Comment:
Nonlinear Poisson's equation example demonstrating the nonlinear diffusion and nonlinear volume force terms.